WordPress/Classes/ViewRelated/Reader/Sorting/ReaderSortingOptionButton.swift

```diff
@@ -0,0 +1,120 @@
+import UIKit
+
+enum ReaderSortingOption: String, CaseIterable {
+    case popularity, date
+    
+    var parameterValue: String {
+        return rawValue
+    }
+    
+    var localizedDescription: String {
+        // TODO: check if strings are localized
+        switch self {
+        case .date:
+            return NSLocalizedString("Recent", comment: "Sorting option description")
+        case .popularity:
+            return NSLocalizedString("Popular", comment: "Sorting option description")
+        }
+    }
+    
+    var image: UIImage {
+        switch self {
+        case .date:
+            // TODO: use proper icon
+            return .gridicon(.calendar)
+        case .popularity:
+            // TODO: use proper icon
+            return .gridicon(.lineGraph)
+        }
+    }
+}
+
+class ReaderSortingOptionButton: UIControl {
```

Comment:
Control that will be used above the cards in the UI to show current sorting option and when user tap it we will show the bottom sheet for picking sorting options.

Reply:
The control has 3 UI elements, an icon view, label and the chevron view. Icon view and label have different content depending the `sortingOption` value